antenna/configlib.py

```diff
@@ -0,0 +1,323 @@
+"""This module contains the configuration infrastructure allowing for deriving
+configuration from a .ini file and the process environment.
+
+In order of precedence:
+
+1. settings overrides (usually only applies when running tests)
+2. process environment
+3. .ini file (specified by ANTENNA_INI environment variable)
+
+Example of usage::
+
+    from antenna.configlib import config
+
+    DEBUG = config('DEBUG', default=True, parser='bool')
```

Comment:
I think `'True'` **looks** so like a mistake or something not very smart. 

The truth is, for booleans in a config system we probably want to do things like converting strings like `'no'`, `'0'` etc to `False`. 

What about simply...:
```
parser = type(default)
if parser is bool:
    value = str(value).lower() not in ('no', '0', 'false')
else:
    value = parser(value)
```
...or something like that.